kernel/uGraph.ml

```diff
@@ -521,7 +522,13 @@ let add_universe vlev strict g =
     in
     let entries = UMap.add vlev (Canonical v) g.entries in
     let g = { entries; index = g.index - 1; n_nodes = g.n_nodes + 1; n_edges = g.n_edges } in
-    insert_edge strict (get_set_arc g) v g
+    insert_edge strict (get_arc g) v g
+
+let add_universe vlev strict g =
+  add_universe_above vlev strict get_set_arc g
```

Comment:
Don't pass a function, pass the actual universe. ie `add_universe_above vlev strict Level.set g`.

Reply:
Done. By the way, is there a way to declare a universe without having an edge on it, since, here, after all, the condition `>= Prop` always holds, so I would assume that we can just declare a node without an edge?

Reply:
I don't understand what you mean. Creating the edge is what ensures the condition holds. It will make it so if we have `foo <= Prop` it will identify with Prop, and it will prevent `Type@{bar} : Prop` which has I have no idea what consequences.

Reply:
Ah, yes, you're right. I was reasoning with a (natively) well-founded graph in mind, but in practice, the algorithm is for any graph, so the constraint is needed.